docs/5-development/04-understanding-hbs-templates.md

```diff
@@ -216,7 +216,7 @@ In this example, even though we're looping over an item from the array, we can s
 
 You can use the following features when writing `.hbs` templates:
 
-### Bindings <a name="syntax_bindings"></a>
+### 4.1 Bindings <a name="syntax_bindings"></a>
```

Comment:
Since the structure of this document so far is to have points and subpoints, I suggest we keep it in the last two points as well. As an alternative, we could also remove the numbering in the whole doc and keep just titles.